Discussion (#50)
MrLongbottom/P9-Rapport: sections/Discussion.tex

\section{Discussion}\label{sec:discussion}
In this section, we analyze and describe the results, we get from running our experiment.

\gls{pr} using our adjacency-matrix successfully improved upon the results of other models in many cases.

> This is worth noting as our adjacency matrix was constructed based on similarity of the documents' topic-distributions, rather than any relation embedded in the documents themselves, such as articles referencing each other as with \cite{yang2009topic}.

Reviewer: ```suggestion
This is worth noting as our adjacency matrix was constructed based on similarity of the documents' topic-distributions, rather than any relation embedded in the documents themselves, such as scientific articles referencing each other as with \cite{yang2009topic}.
```

This means that our initial idea of using the topic distributions as a similarity measure in the \gls{pr} algorithm, was a success.

> They use the standard \gls{pr} in $\frac{4}{13}$ test cases, where we use \gls{pr} $\frac{12}{18}$.

Reviewer: ```suggestion
They use the standard \gls{pr} in $\frac{4}{13}$ test cases, where we use \gls{pr} in $\frac{12}{18}$ test cases.
```

In \cite{yang2009topic}, they use a specialized version of \gls{pr} which is their focus.
We direct our attention to whether a standard \gls{pr} could improve \gls{ir} results.
\todo[inline]{Vurdering af egen adjacency matrix som edges i PR i forhold til original paper. Hvis vi når det: generaliserbarhed af dette (andet datasæt)}

\gls{lm} performed surprisingly bad compared to the results of \cite{yang2009topic}. 
\todo[inline]{We need to figure out what we are going to say about lm's bad performance, or maybe move some of it to appendix if we want to avoid talking too much about it}

> \gls{lda} gets good topic query retrieval results when combined with \gls{pr}, as seen on \autoref{tab:results}.

Reviewer: ```suggestion
\gls{lda} gets good topic query retrieval results when combined with \gls{pr}, as seen in \autoref{tab:results}.
```

This comes at the cost of very bad document query retrieval results.

> However, \autoref{tab:results_precision_at_10} shows that \gls{lda} only gets the best precision@10 and precision@100 with queries of 4 words.

Reviewer 2: ```suggestion
Though \gls{lda} has good \gls{map} results, \autoref{tab:results_precision_at_10} shows that \gls{lda} only gets the best precision@10 and precision@100 with queries of 4 words.
```

> This means that while it is well suited for ranking a large number of documents fairly well (as with \gls{map}), it is not finding a relatively small number of relevant documents with great certainty.

Reviewer 2: ```suggestion
This means that while it is well suited for ranking a large number of documents fairly well (as with \gls{map}), it is not good at finding a relatively small number of relevant documents
```

It also points towards \gls{lda} being more reliant on a larger amount of context than many of the other models, and could therefore likely benefit greatly from query expansion methods.
However, for this experiment \gls{lda}'s purpose is mostly to improve the topic retrieval performance of other models it is combined with, at the cost of reducing their document retrieval performance.

> Overall, the most successful combinations of baselines for combining document query retrieval and topic query retrieval seems to be $\gls{bm25}+\gls{pr}$ and $\gls{bm25}*\gls{pr}$.

Reviewer: ```suggestion
Overall, the most successful combinations of baselines for combining document query retrieval and topic query retrieval seem to be $\gls{bm25}+\gls{pr}$ and $\gls{bm25}*\gls{pr}$.
```
